Coroutines/declarators.tex

```diff
@@ -149,6 +149,13 @@
  \mbox{\tcode{$p$.get_return_object()}}. A call to a \tcode{get_return_object} is sequenced before the call to \linebreak
 \mbox{\tcode{initial_suspend}} and is invoked at most once.
 
+\pnum
+When the coroutine reaches \textit{final suspend point} the coroutine is suspended and the expression \linebreak
+\tcode{$p$.final_suspend(std::experimental::coroutine_handle<$P$>::from_promise($p$))} is evaluated.
+The result of this expression must have type \tcode{std::experimental::coroutine_handle<}$Z$\tcode{>} for some type $Z$.
+The coroutine identified by the returned handle is resumed as if by calling the \tcode{resume()} method. 
```

Comment:
@GorNishanov Does this section need some extra wording to require implementations to perform a tail-call here?

Reply:
Yes, Grab the wording from [await.expr]